src/containers/FormikForm/FormikMetaTagSearch.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Fragment, useState } from 'react';
+import { injectT } from '@ndla/i18n';
+import { DropdownInput } from '@ndla/forms';
+import { AsyncDropdown } from '../../components/Dropdown';
+import { fetchSearchTags } from '../../modules/draft/draftApi';
+
+interface target {
+  target: {
+    name: string;
+    value: string[] | null;
+  };
+}
+
+interface props {
+  t(lang: string): void;
+  locale: string;
+  initTags: string[];
+  field: {
+    onChange: (target: target) => void;
+    name: string;
+  };
+  form: {
+    setFieldTouched: (fieldName: string, a: boolean, b: boolean) => void;
+  };
+}
+
+interface asyncDropdownProps {
+  selectedItems: (tags: string[]) => tagWithTitle[];
+  removeItem: (tag: string) => void;
+  inpProps: any;
+}
+
+interface tagWithTitle {
+  title: string;
+}
+
+const FormikMetaTagSearch = ({ t, locale, initTags, field, form }: props) => {
+  const convertToTagsWithTitle = (tagsWithoutTitle: string[]): object[] => {
+    return tagsWithoutTitle.map(c => ({ title: c }));
+  };
+
+  const [tags, setTags] = useState(initTags);
+
+  const searchForTags = async (inp: string) => {
+    const result = await fetchSearchTags(inp, locale);
+    result.results = convertToTagsWithTitle(result.results);
+    return result;
```

Comment:
Egentlig bare en mening, men jeg liker å ikke mutere på objekter :)
```suggestion
    return {
      ...result,
      results: convertToTagsWithTitle(result.results),
    };
```